src/NovelRT/Windowing/Glfw/GlfwWindowingDevice.cpp

```diff
@@ -92,6 +92,11 @@ namespace NovelRT::Windowing::Glfw
         {
             EngineConfig::RequiredVulkanInstanceExtensions().emplace_back(extensions[i]);
         }
+
+        // As of VulkanSDK 1.3.216, we need to opt-in to non-conformant drivers otherwise we'll get Error -9:
+        // VK_ERROR_INCOMPATIBLE_DRIVER - This affects MoltenVK
+        EngineConfig::RequiredVulkanInstanceExtensions().emplace_back("VK_KHR_portability_enumeration");
+        EngineConfig::RequiredVulkanInstanceExtensions().emplace_back("VK_KHR_get_physical_device_properties2");
```

Comment:
Why not just take a `T&` to the vector as a variable and then call emplace back on that? Fetching the reference to the vector twice seems weird.

Reply:
If so, can we do that prior to the for-loop? Then the for-loop could use the `T&` instead of making this call.. possibly 10+ times

Reply:
That sounds like a good optimisation to me. Go for it.